README.md

```diff
@@ -38,90 +38,133 @@
+StellarGraph is built on [TensorFlow 2](https://tensorflow.org/) and its [Keras high-level API](https://www.tensorflow.org/guide/keras), as well as [Pandas](https://pandas.pydata.org) and [NumPy](https://www.numpy.org). It is thus user-friendly, modular and extensible. It interoperates smoothly with code that builds on these, such as the standard Keras layers and [scikit-learn](http://scikit-learn.github.io/stable), so it is easy to augment the core graph machine learning algorithms provided by StellarGraph.
 
+## Getting Started
 
-## Guiding Principles
+[The numerous detailed and narrated examples](https://github.com/stellargraph/stellargraph/tree/master/demos/) are a good way to get started with StellarGraph. There is likely to be one that is similar to your data or your problem (if not, [let us know](#getting-help)).
 
-StellarGraph uses the [Keras](https://keras.io/) API as implemented in the [TensorFlow](https://tensorflow.org/) library and adheres to the same
-guiding principles as Keras: user-friendliness, modularity, and easy extendability. Modules and layers
-of StellarGraph library are designed so that they can be used together with
-standard Keras layers and modules, if required. This enables flexibility in using existing
-or creating new models and workflows for machine learning on graphs.
+To run these examples, you'll need to install StellarGraph. One of the following might be appropriate:
 
-## Getting Started
+- Using pip: `pip install stellargraph[demos]`
+- Using conda: `conda install -c stellargraph stellargraph`
+
+(See [Installation](#Installation) section for more details and more options.)
+
+## Getting Help
+
+If you get stuck or have a problem, there's many ways to make progress and get help or support:
+
+- [Read the documentation](https://stellargraph.readthedocs.io).
+- [Consult the examples](https://github.com/stellargraph/stellargraph/tree/master/demos/)
+- Contact us:
+  - [Ask questions and discuss problems on the StellarGraph Discourse forum](https://community.stellargraph.io)
+  - [File an issue](https://github.com/stellargraph/stellargraph/issues/new/choose)
+
+
+## Example: GCN
+
+One of the earliest deep machine learning algorithms for graphs is a Graph Convolution Network (GCN) [6]. It's easy to apply with StellarGraph. The follow example shows classifying nodes into some number of ground-truth classes, based on node and edge data loaded from files, with Pandas.
+
+``` python
+import tensorflow as tf
+import stellargraph as sg
+import pandas as pd
+from sklearn import model_selection
+
+#### Data preparation ####
+
+# Use Pandas to load and preprocess the data
+nodes = pd.read_csv("nodes.csv") # columns of node data
+edges = pd.read_csv("edges.csv") # "source" & "target" columns for each link between nodes
+
+node_classes = pd.read_csv("labels.csv") # a column of each node's class/label
+targets = pd.get_dummies(node_classes) # one-hot encoding
```

Comment:
Thoughts on making this smaller?
```suggestion
# load data into pandas dataframes
nodes, edges, targets = load_my_data()
```

Reply:
Why not simplify by using one of the existing datasets in `sg.datasets`?

Reply:
I was trying to have this be more of a StellarGraph workflow taster, than showing of GCN in particular. In particular, I was trying to make it easy for a data scientist to see how their data fits into this, by showing that the "load the data" step is just normal Pandas. The built-in datasets package up all of that data prep to keep our algorithm demos focused on the algorithms, but my motivation with this was a bit broader.

I think I'll go with Kevin's suggestion, although tweaking the comment slightly.